src/01-functions.js

```diff
@@ -112,7 +131,17 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+if (typeof a === typeof b) {
+  return true;
+}
+  else {
```

Comment:
Let's watch our indentation and unneeded space after line 136.  Also if we do not ` return true ` on line 136 - do we need an ` else ` at all?